src/components/Label/Label.component.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import styles from './Label.module.scss';
+
+const Label = ({ required, children, className, ...props }) => {
+  return (
+    <label className={classnames(styles.label, className)} {...props}>
+      {!!required && <span className={styles.label_require}>* </span>}
+      {children}
+    </label>
+  );
+};
```

Comment:
Do we really need this? I thought Ant Design have the required stylings for the form item. And we'd better reuse them. Please refer to the contact-us form on how to address the same at https://github.com/pingcap/tug-website/blob/main/src/pages/contact-us/incident/form/form.fields.js

<img width="572" alt="Screen Shot 2021-09-10 at 8 32 32 PM" src="https://user-images.githubusercontent.com/123074/132825210-9b6ca5e4-0a41-4107-8a48-437288429e0a.png">